### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by ghost**
- [DEMO LINK](https://bgutai.github.io/layout_calendar/)
- [TEST REPORT LINK](https://bgutai.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,66 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--mon calendar--month-length--31">
+      <div class="calendar__day"></div>
+
```
Comment (roman-mirzoian) [inline src/index.html]: Do not leave unnecessary empty lines.

**Review by roman-mirzoian — CHANGES_REQUESTED** [review]
Everything is cool, but now if you set the modifier to 31 days, it will show 30.
image

**Review by alexander-ignatow — CHANGES_REQUESTED** [review]
Looks good, but you still have one bug:
The calendar cannot show 31 days (when I apply style `calendar--month-length--31` it still shows 30 days)

Hint: the error is with HTML, not styles

**Review by Moroz-Dmytro — CHANGES_REQUESTED** [review]
Cell is without content(31)
image

**Review by alexander-ignatow — APPROVED** [review]
LGTM 👍 

Next time pay more attention and make sure to smoke test your solution before submitting 😉

**Pull request closed without merging**